lib/rest/routes/credentials.js

```diff
@@ -98,13 +98,22 @@ module.exports = function () {
   });
 
   router.get('/:consumerId', function (req, res, next) {
-    credentialSrv.getAllCredentials(req.params.consumerId)
-    .then(credentials => {
-      res.json({credentials});
-    })
-    .catch(err => {
-      next(err);
-    });
+    const {params, query} = req;
```

Comment:
Nit: I would not de-structure this — it biases the view that this stuff is coming from the request.